editor/block-toolbar/index.js

```diff
@@ -19,19 +20,51 @@ import './style.scss';
 import BlockSwitcher from '../block-switcher';
 import BlockMover from '../block-mover';
 import BlockRightMenu from '../block-settings-menu';
+import { isMac } from '../utils/dom';
+
+/**
+ * Module Constants
+ */
+const { ESCAPE, F10 } = keycodes;
 
 function FirstChild( { children } ) {
 	const childrenArray = Children.toArray( children );
 	return childrenArray[ 0 ] || null;
 }
 
+function metaKeyPressed( event ) {
```

Comment:
Would it have been possible to use the `KeyboardShortcuts` component to manage and normalize these keyboard combinations?

https://github.com/WordPress/gutenberg/tree/master/components/keyboard-shortcuts

Reply:
I tried but it failed to trigger on "command" key only, not sure why

Reply:
>I tried but it failed to trigger on "command" key only, not sure why

Might have been running into caveat of Mousetrap not firing callbacks while input is focused. See #3031 for workaround.